### PR TITLE
First batch of controller refactoring

### PR DESCRIPTION
**Pull request by zaparka**
Rubocop refactoring. Ready to merge.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,11 +2,19 @@ class ApplicationController < ActionController::Base
   WillPaginate.per_page = 50
 
   def after_sign_in_path_for(resource)
```
Comment (leoapost) [inline app/controllers/application_controller.rb]: how about:

```ruby
def after_sign_in_path_for(resource)
  return admin_path if resource

  root_url
end
```

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -2,11 +2,19 @@ class ApplicationController < ActionController::Base
...
-    admin_path
+    if resource
+      admin_path
+    else
+      root_url
+    end
   end
 
   def after_sign_out_path_for(resource)
```
Comment (leoapost) [inline app/controllers/application_controller.rb]: how about:

```ruby
def after_sign_out_path_for(resource)
  return new_user_session_path if resource

  root_url
end
```

File: app/controllers/courts_controller.rb [hunk 3]
```diff
@@ -65,61 +48,15 @@ def show
...
       end
     end
   end
 
   private
-  def find_court
-    @court = Court.friendly.find(params[:id])
-  end
 
-  def courts_csv
```
Comment (leoapost) [inline app/controllers/courts_controller.rb]: isn't this needed anymore?

Comment (zaparka) [inline app/controllers/courts_controller.rb]: @leoapost  Well, the tests are commented out. So I assume that it isn't. I can ask Dan just to be sure.

**Pull request merged**